powertools-idempotency/src/main/java/software/amazon/lambda/powertools/idempotency/persistence/BasePersistenceStore.java

```diff
@@ -34,12 +31,8 @@
 import java.security.NoSuchAlgorithmException;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
-import java.util.Map;
-import java.util.OptionalLong;
-import java.util.OptionalInt;
+import java.util.*;
```

Comment:
I think our standard is no * in non test code.

Reply:
automatic by the IDE... fixed